workbench/test/test_filethumbs.py

```diff
@@ -55,6 +56,7 @@ def test_three_thumbs_initial_state(self):
             initial_up = int(up_count.text)
             initial_down = int(down_count.text)
             thumb.find_element_by_css_selector('span.upvote').click()
+            time.sleep(5)  # this is a dirty, dirty hack. Feel free to remove if you're working in this code.
```

Comment:
This (bok choy) test isn't waiting long enough for the event to finish doing things, so I figured 5 seconds on every test build is a fair price to pay for non-flakiness in a rarely-updated repository.